presto-hive/src/test/java/io/prestosql/plugin/hive/metastore/TestMetastoreUtil.java

```diff
@@ -153,4 +156,12 @@ public void testHiveSchemaPartition()
         Properties actual = MetastoreUtil.getHiveSchema(ThriftMetastoreUtil.fromMetastoreApiPartition(TEST_PARTITION_WITH_UNSUPPORTED_FIELDS), ThriftMetastoreUtil.fromMetastoreApiTable(TEST_TABLE_WITH_UNSUPPORTED_FIELDS, TEST_SCHEMA));
         assertEquals(actual, expected);
     }
+
+    @Test
+    public void testTupleDomainToString()
+            throws Exception
+    {
+        Optional<List<String>> ds = MetastoreUtil.partitionKeyFilterToStringList(Arrays.asList("ds1", "ds2", "ds3"), TupleDomain.all(), false);
+        System.err.println(ds.get());
```

Comment:
Remove the printing. Probably add an assertion or remove the entire test method.

Reply:
oops again, yea i'll assert or remove it